man/coredumpctl.xml

```diff
@@ -217,6 +218,15 @@
         will be used. </para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>-A</option> <replaceable>ARGS</replaceable></term>
+        <term><option>--debugger-arguments=</option><replaceable>ARGS</replaceable></term>
+
+        <listitem><para>Pass the given <replaceable>ARGS</replaceable> as command line arguments
+        to the debugger. Quote as appropriate when <replaceable>ARGS</replaceable> contain whitespace.
```

Comment:
I'd be nice to say that those are **extra** arguments. After all, we pass a bunch ourselves. But maybe the example is enough.

Reply:
Added the "extra".